plutus-contract/test/Spec/golden/traceOutput - pubKeyTransactions.txt

```diff
@@ -1,83 +1,83 @@
-Slot 00000: TxnValidate 0636250aef275497b4f3807d661a299e34e53e5ad3bc1110e43d1f3420bc8fae
+Slot 00000: TxnValidate 686106f51262efb810d99311d3bbe478ba5abb92b5be2e1f9488d56622caf6f4
 Slot 00000: SlotAdd Slot 1
-Slot 00001: W1: InsertionSuccess: New tip is Tip(slot= Slot 1, blockId= BlockId(-8798726201061206542), blockNo= 0). UTxO state was added to the end.
-Slot 00001: W2: InsertionSuccess: New tip is Tip(slot= Slot 1, blockId= BlockId(-8798726201061206542), blockNo= 0). UTxO state was added to the end.
-Slot 00001: W3: InsertionSuccess: New tip is Tip(slot= Slot 1, blockId= BlockId(-8798726201061206542), blockNo= 0). UTxO state was added to the end.
-Slot 00001: W4: InsertionSuccess: New tip is Tip(slot= Slot 1, blockId= BlockId(-8798726201061206542), blockNo= 0). UTxO state was added to the end.
-Slot 00001: W5: InsertionSuccess: New tip is Tip(slot= Slot 1, blockId= BlockId(-8798726201061206542), blockNo= 0). UTxO state was added to the end.
-Slot 00001: W6: InsertionSuccess: New tip is Tip(slot= Slot 1, blockId= BlockId(-8798726201061206542), blockNo= 0). UTxO state was added to the end.
-Slot 00001: W7: InsertionSuccess: New tip is Tip(slot= Slot 1, blockId= BlockId(-8798726201061206542), blockNo= 0). UTxO state was added to the end.
-Slot 00001: W8: InsertionSuccess: New tip is Tip(slot= Slot 1, blockId= BlockId(-8798726201061206542), blockNo= 0). UTxO state was added to the end.
-Slot 00001: W9: InsertionSuccess: New tip is Tip(slot= Slot 1, blockId= BlockId(-8798726201061206542), blockNo= 0). UTxO state was added to the end.
-Slot 00001: W10: InsertionSuccess: New tip is Tip(slot= Slot 1, blockId= BlockId(-8798726201061206542), blockNo= 0). UTxO state was added to the end.
-Slot 00001: W1: TxSubmit: da6b9c63ed30d9d469047e8dae81971f3dd7dcb478385fb1d442a811eea1caa1
-Slot 00001: TxnValidate da6b9c63ed30d9d469047e8dae81971f3dd7dcb478385fb1d442a811eea1caa1
+Slot 00001: W2e463c80f68b7047fa67403b1911f161aa597bbdf683d9b4e3f66ef3762c1499269f4f5278db3368fe58905f4cf581642fe40ead0506e7863c5a543254d21876070685fdff668e697efca70098985dec1497e079f958296a335b47ab3d0dffa1b394f26269b9e620c07148bfe1218a076be87eec941ca20340b443575467ef04: InsertionSuccess: New tip is Tip(slot= Slot 1, blockId= BlockId(5694917818900844593), blockNo= 0). UTxO state was added to the end.
```

Comment:
Maybe we could shorten the key when it's displayed here.

Reply:
Good idea, done!